sidecar/sokuji_sidecar/gpt_sovits/text_splitter.py

```
@@ -0,0 +1,125 @@
+        初始化文本切分器。
+
+        :param max_len: 软限制最大长度 (Effective Length)。超过此长度遇到分隔符时会切分。
+        :param min_len: 硬限制最小长度 (Effective Length)。小于此长度遇到终止符也不会切分。
+        """
+        self.max_len: int = max_len
+        self.min_len: int = min_len
+
+        # 1. 定义基础字符集合
+        # 只要标点块中包含这些字符，就视为 Ending (终止符)
+        self.end_chars: Set[str] = {
+            '。', '！', '？', '…',
```

Ruff (0.15.21)

[warning] String contains ambiguous `！` (FULLWIDTH EXCLAMATION MARK). Did you mean `!` (EXCLAMATION MARK)?

(RUF001)